media/css/mozorg/contact-spaces.scss

```diff
@@ -16,86 +20,42 @@
          url('/media/fonts/icons-contact.woff') format('woff');
 }
 
-$lightGray: lighten(#e7e5e2, 6%);
-
-a:link,
-a:visited {
-    color: $color-link;
-}
-
-a:hover,
-a:active,
-a:focus {
-    color: darken($color-link, 10%);
-}
-
-// Add a double-arrow after 'go' links
-.go {
```

Comment:
nit: `class="go"` could be removed from a lot of templates in contact/communities.